src/python-common/ceph/deployment/service_spec.py

```diff
@@ -863,6 +863,7 @@ def __init__(self,
                  rgw_frontend_port: Optional[int] = None,
                  rgw_frontend_ssl_certificate: Optional[List[str]] = None,
                  rgw_frontend_type: Optional[str] = None,
+                 rgw_frontend_extra_args: Optional[str] = None,
```

Comment:
> WDYT about having this as a list of str vs. just a str? For example, if I wanted to specify multiple params the format would be
> 
> ```
> rgw_frontend_extra_args:
> - "tcp_nodelay=0"
> - "max_header_size=65536"
> ```
> 
> and we'd just put them in space separated into the config option vs. the current
> 
> ```
> rgw_frontend_extra_args: "tcp_nodelay=0 max_header_size=65536"
> ```
> 
> We've used both before. `extra_container_args` uses the list format while `trusted_ip_list` for iscsi uses the string format. Both are okay, although I have a personal preference for the list. Thoughts?

Well, since in this case all the options are in the form `opt=value` I think your proposal makes a lot of sense and I like the idea of having the list because it's cleaner in terms of syntax. I'll adjust the code to give it a try :+1:

Reply:
I changed the code to use a list :+1:  and updated the testing section with the results after the changes as well.